.travis.yml

```diff
@@ -1,3 +1,8 @@
 language: java
 jdk:
-  - openjdk8
\ No newline at end of file
+  - openjdk8
+deploy:
+  provider: heroku
+  api-key:
+    secure: $HEROKU_API_KEY
```

Comment:
Where does the heroku api key come from?

Reply:
It is put into Travis CI.

Reply:
It comes from Heroku